internal/cli/ex-get-slots.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/coreos/airlock/internal/lock"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// cmdGetSlots holds `airlock ex get slots`
+	cmdGetSlots = &cobra.Command{
+		Use:   "slots",
+		Short: "Introspect groups/slots state",
+		RunE:  runGetSlots,
+	}
+)
+
+// runGetSlots performs live introspection of groups/slots.
+func runGetSlots(cmd *cobra.Command, cmdArgs []string) error {
+	if runSettings == nil {
+		return errors.New("nil runSettings")
+	}
+
+	for group, maxSlots := range runSettings.LockGroups {
+		ctx, cancel := context.WithTimeout(context.Background(), runSettings.EtcdTxnTimeout)
+		defer cancel()
+
+		manager, err := lock.NewManager(ctx, runSettings.EtcdEndpoints, group, maxSlots)
+		if err != nil {
+			return err
+		}
+		semaphore, err := manager.FetchSemaphore(ctx)
+		if err != nil {
+			return err
+		}
+		printHumanShort(group, semaphore)
+	}
+
+	return nil
+}
+
+// printHumanShort prints groups/slots details in a short, human-friendly way.
+func printHumanShort(group string, semaphore *lock.Semaphore) {
+	if group == "" || semaphore == nil {
+		return
+	}
+
+	fmt.Printf("group: %s\n", group)
+	fmt.Printf(" semaphore slots: %d\n", semaphore.TotalSlots)
+	fmt.Printf(" lock owners:\n")
+	for _, uuid := range semaphore.Holders {
+		fmt.Printf(" - %s\n", uuid)
+	}
+	fmt.Printf("\n---\n")
```

Comment:
Should this print be moved outside of `printHumanShort` where we could skip printing it after the last `group`?

Reply:
Maybe. I don't have strong ideas on this. The current output looks like the following, with the `---` here just meant as a visual clue to delimit a group even when empty:
```
group: workers
 semaphore slots: 2
 lock owners:
 - c988d2509fdf4cdcbed39037c56406fb

---
group: masters
 semaphore slots: 1
 lock owners:

---
group: default
 semaphore slots: 2
 lock owners:

---
```